resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/submit/KubernetesClientUtils.scala

```diff
@@ -29,13 +29,16 @@ import io.fabric8.kubernetes.api.model.{ConfigMap, ConfigMapBuilder, KeyToPath}
 
 import org.apache.spark.SparkConf
 import org.apache.spark.deploy.k8s.{Config, Constants, KubernetesUtils}
+import org.apache.spark.deploy.k8s.Config.KUBERNETES_LABEL_MAX_LENGTH
 import org.apache.spark.deploy.k8s.Constants.ENV_SPARK_CONF_DIR
 import org.apache.spark.internal.Logging
 
 private[spark] object KubernetesClientUtils extends Logging {
 
   // Config map name can be 63 chars at max.
-  def configMapName(prefix: String): String = s"${prefix.take(54)}-conf-map"
+  def configMapName(prefix: String, suffix: String = "-conf-map"): String = {
```

Comment:
Do we have any place using `suffix` parameter?

Reply:
OK, I will revert this part

Reply:
The `suffix` is different across Spark codebase actually, but the same for the callers of this method currently ... We may need another PR to reflect other places to call the method here for the consistency 
 of config map names